test/functional/apps/settings/_index_pattern_results_sort.js

```diff
@@ -93,9 +93,11 @@ define(function (require) {
         });
 
         bdd.it('makelogs data should have expected number of fields', function () {
-          return settingsPage.getFieldsTabCount()
-          .then(function (tabCount) {
-            expect(tabCount).to.be('' + expectedFieldCount);
+          return common.tryForTime(5000, function () {
```

Comment:
I get that, and I have no problem doing that. I would just like the tryForTime to be *inside* of `getFieldsTabCount`, not in the test code.